core/unit_test/TestTeamBasic.hpp

```diff
@@ -178,17 +178,14 @@ TEST(TEST_CATEGORY, large_team_scratch_size) {
   // problems allocating too much memory.
   const size_t per_team_extent = 268435460;
 #else
```

Comment:
You should make this non-const since you (potentially) modify it after
```suggestion
  // problems allocating too much memory.
  size_t per_team_extent = 268435460;
#else
```

Reply:
Fixed!